logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/GetSessionsUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.wire.kalium.logic.feature.auth
+
+import com.wire.kalium.logic.data.session.SessionRepository
+
+class GetSessionsUseCase(private val sessionRepository: SessionRepository) {
+
+    suspend operator fun invoke(): List<AuthSession> = sessionRepository.getSessions() //TODO Any kind of failure possible?
```

Comment:
Might contain expired sessions? But I guess this will generate an error later when there's an attempt to use it.

Reply:
Could be. 

I didn't plug in session expiration yet.

How that will work is up to us I guess. Maybe we can filter it out and call it `GetValidSessionsUseCase`, or remove them a
completely once we detect they're expired.